databricks/koalas/frame.py

```diff
@@ -27,6 +27,7 @@
 from pyspark.sql.types import StructType, to_arrow_type
 from pyspark.sql.utils import AnalysisException
 
+from databricks import koalas as ks
```

Comment:
add comment explaining why we are importing koalas as ks

Reply:
yup